fml/mapping.h

```diff
@@ -63,6 +64,22 @@ class FileMapping : public Mapping {
   FXL_DISALLOW_COPY_AND_ASSIGN(FileMapping);
 };
 
+class DataMapping : public Mapping {
```

Comment:
Where is this used?

Reply:
Working with asset bundles and such on Fuchsia.